files/build_templates/systemd_regex.json

```diff
@@ -6,7 +6,7 @@
     },
     {
         "tag": "watchdog-timeout",
-        "regex": "(?:watchdog|Watchdog) timeout .limit.([0-9])min.",
+        "regex": ".*(?:watchdog|Watchdog) timeout .limit.([0-9])min.",
```

Comment:
Is there an example message for this regex?
If the timeout is 20 minutes, this regex seems not work: ([0-9])min

Reply:
Jul  2 03:49:13 BLZ22-0101-0124-02T0 systemd[1]: systemd-journald.service: Watchdog timeout (limit 3min)!